src/containers/AllComments.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Comments from '../components/comments/Comments';
+import { reset } from '../action/common/reset';
+import { getComments } from '../selectors/comments/comments';
+import { fetchComments } from '../action/comments';
+import { withRouter } from 'react-router-dom';
```

Comment:
```suggestion
import { withRouter } from 'react-router-dom';

```
Keep your whitespace consistent across files.